deploy/examples/csi/rbd/storageclass-radosnamespace.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: ceph.rook.io/v1
```

Comment:
The only concern of this new storageclass is that we need to take care of updating this as well if we add/remove any new fields for rbd specific option for rbd storageclass , this is kind of a maintenance burden we will get :(

Reply:
I share this concern. Though I'm on the fence about whether to keep the example. At the end of the day, though, I think the only truly unique thing here is rados namespace and storageclass's `clusterID`. I think the documentation is sufficiently clear enough that users shouldn't require an example manifest to understand, and maybe it would be best to remove this so it doesn't grow stale over time.

Reply:
@BlaineEXE  wrote: 

> I share this concern. Though I'm on the fence about whether to keep the example. At the end of the day, though, I think the only truly unique thing here is rados namespace and storageclass's `clusterID`. I think the documentation is sufficiently clear enough that users shouldn't require an example manifest to understand, and maybe it would be best to remove this so it doesn't grow stale over time.

Agreed. I can remove the example manifest from the PR.

Reply:
@BlaineEXE example manifest is now removed.

Reply:
I do think the full example manifests are worth the maintenance burden for common scenarios. But I'm fine skipping this example for now since it's less common.